src/invoices/InvoiceForm/InvoiceForm.test.js

```diff
@@ -126,6 +126,7 @@ describe('InvoiceForm component', () => {
     const accountLabel = `${accountNo} (${appSystemNo})`;
 
     expect(container.querySelector('#selected-accounting-code-selection-item')).toHaveTextContent('');
+    await userEvent.click(screen.getAllByText('stripes-components.selection.controlLabel')[4]);
```

Comment:
Is there another way to bind to an element other than the index in an array of elements of the same type? This is a fairly brittle method that can break tests when new elements are added.

Applicable for all places in this file

Reply:
And this not resolved as well. Please use specific anchors for selections like `screen.getByRole('button', { name: <name> })`